### PR TITLE
feat(script): fix lightClient issue and add verification scripts

### PR DESCRIPTION
**Pull request by cosinlink**
### Description

fix lightClient issue, add verification scripts and cross-chain operation scripts

### Changes

Notable changes:
* update readme
* add verify scan scripts
* optimize deploy script
* fix lightclient issue, sync code to v0.0.7
* add cross-chain operation scripts and docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,22 @@ make build
 npm run deploy:test
 ```
 
+## Verify on BSCScan
+```shell
+npm run deploy:testnet
```
[the remark this answers is not in the document]
Comment (SolidityGo) [reply README.md]: fixed

[the remark this answers is not in the document]
Comment (cosinlink) [reply README.md]: fixed

File: foundry-scripts/TokenHub.s.sol [hunk 2]
```diff
@@ -1,28 +1,36 @@
...
 
-import "forge-std/Script.sol";
-
-import "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
-import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
-
-import "../contracts/GnfdProxy.sol";
-import "../contracts/GnfdProxyAdmin.sol";
-import "../contracts/GnfdLightClient.sol";
+import "../contracts/Deployer.sol";
```
Comment (guagualvcha) [inline foundry-scripts/TokenHub.s.sol:3]: I don't think this is a good dependency cycle.

**Review by guagualvcha — CHANGES_REQUESTED** [review]
we need a refined command system, not only transfer out command.

**Comment by guagualvcha** [comment]
Need to improve the readme

**Review by guagualvcha — APPROVED** [review]

**Pull request merged**